alerting/src/main/kotlin/org/opensearch/alerting/model/Monitor.kt

```diff
@@ -77,6 +79,21 @@ data class Monitor(
                     require(trigger is DocumentLevelTrigger) { "Incompatible trigger [${trigger.id}] for monitor type [$monitorType]" }
             }
         }
+
+        // validate the number of Actions Across Triggers
+        if (MonitorRunnerService.monitorCtx.totalMaxActionsAcrossTriggers > AlertingSettings.UNBOUNDED_ACTIONS_ACROSS_TRIGGERS) {
```

Comment:
If we don't want to bloat the Monitor's `init` with this, since it's depending on `MonitorRunnerService`'s monitor ctx which is runtime info, we can just put these validations in the `TransportIndexMonitorAction` logic.